our-app/src/App.css

```diff
@@ -14,7 +14,7 @@
 }
 
 .App-header {
-  background-color: #282c34;
+  background-color:#c9af98;
```

Comment:
This background color really doesn't fit the theme of the app.. would any shades of grey work? Thoughts?

Reply:
I tried gray and I personally think it looks quite dull but I would want you to personally visualize it.

Reply:
Did you want me to remove the first image? That is the only image with description and text in it.